packages/webapp/src/components/Navigation/TopMenu/TopMenu.jsx

```diff
@@ -221,19 +222,15 @@ const TopMenu = ({ history, isMobile, showNavigation, onClickBurger }) => {
     </>
   );
 
-  const Logo = ({ withoutWords, withLink, ...props }) => {
-    return withoutWords ? (
-      <IconLogo
-        alt="LiteFarm Logo"
-        className={clsx(styles.logo, withLink && styles.withLink)}
-        {...props}
-      />
-    ) : (
-      <WordsLogo
-        alt="LiteFarm Logo"
-        className={clsx(styles.logo, withLink && styles.withLink, styles.paddingTopBottom)}
-        {...props}
-      />
+  const Logo = ({ withoutWords, onClick }) => {
+    if (!withoutWords) {
```

Comment:
The double negation is a bit confusing here, could we have this prop be `withWords` instead, and then we could just do `if (withWords) {...`?

Alternatively we could do `if (withoutWords)` as the first block here.

Reply:
I confused myself too... I went with the second solution, thank you!